src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/bulkscan/validation/in/OcrDataField.java

```diff
@@ -1,25 +1,24 @@
-package uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan;
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.validation.in;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.Data;
+import lombok.Getter;
 
-@Data
+@Getter
 public class OcrDataField {
```

Comment:
->lib

Reply:
Moved into the BSP Commons Lib as requested